okta/helpers.py

```diff
@@ -39,6 +39,7 @@ def to_lower_camel_case(string):
     """
     components = string.split('_')
     # lower first letter in the first component
-    components[0] = components[0][0].lower() + components[0][1:]
+    if components[0]:
+        components[0] = components[0][0].lower() + components[0][1:]
```

Comment:
Do we need to check components' length before accessing?

Reply:
No: https://docs.python.org/3.8/library/stdtypes.html#truth-value-testing